CordovaLib/Classes/Public/CDVViewController.m

```diff
@@ -304,45 +301,12 @@ - (void)viewDidLoad
         [CDVTimer stop:@"TotalPluginStartup"];
     }
 
-    // /////////////////
-    NSURL* appURL = [self appUrl];
-    __weak __typeof__(self) weakSelf = self;
-
-    [CDVUserAgentUtil acquireLock:^(NSInteger lockToken) {
-        // Fix the memory leak caused by the strong reference.
-        [weakSelf setLockToken:lockToken];
-        if (appURL) {
```

Comment:
I don't think the logic inside the acquireLock block should be removed, it seems to handle the case where the appUrl is invalid and loads an error page instead